Source/Krypton Toolkit/Shared/Krypton.Toolkit.Suite.Extended.Language.Model/Classes/ButtonLanguageController.cs

```diff
@@ -0,0 +1,126 @@
+﻿namespace Krypton.Toolkit.Suite.Extended.Language.Model
+{
+    public class ButtonLanguageController
+    {
+        #region Variables
+        private static string[] _buttonTextArray = new string[6];
+        #endregion
+
+        #region Constructor
+        /// <summary>Initializes a new instance of the <see cref="ButtonLanguageController"/> class.</summary>
+        public ButtonLanguageController()
+        {
+
+        }
+        #endregion
+
+        #region Methods
+        /// <summary>Sets the language.</summary>
+        /// <param name="language">The language.</param>
+        /// <param name="okText">The ok text.</param>
+        /// <param name="yesText">The yes text.</param>
+        /// <param name="noText">The no text.</param>
+        /// <param name="cancelText">The cancel text.</param>
+        /// <param name="abortText">The abort text.</param>
+        /// <param name="ignoreText">The ignore text.</param>
+        private static void SetLanguage(SelectedLanguage language, string okText = "", string yesText = "", string noText = "", string cancelText = "", string abortText = "", string ignoreText = "")
+        {
+            switch (language)
```

Comment:
Should these also have the equivalent shortcut chars set as well ?

Reply:
I don't think so, as the optional parameters are there for other languages that are not supported by the language manager.